docs/UserGuide.md

```diff
@@ -402,8 +402,9 @@ Format: `export DIRPATH [-o FILENAME]`
 </div>
 
 Examples:
-* `export c:/mcgymmy` exports the save file as `mcgymmy.json` to `c:/mcgymmy`
-* `export c:/mcgymmy -o save_file` exports the save file as `save_file.json` to `c:/mcgymmy`
+* For Windows users:`export c:/mcgymmy` exports the save file as `mcgymmy.json` to `c:/mcgymmy`
+* For Windows users:`export c:/mcgymmy -o save_file` exports the save file as `save_file.json` to `c:/mcgymmy`
+* For Mac users: `export /Users/<username>/Desktop/save_file.json` exports `save_file.json` to the Desktop from McGymmy.
```

Comment:
The save_file.json should be omitted. The Syntax is similar to the one in windows except the folder structure is different ><

Reply:
ok changed!